File: testcases/paper/FilterMethodCalls/FilterMethodCalls.py
class ExternalClass:
    def External(self):
        return 5

class FilterMethodCalls:
    def __init__(self):
        self.i = None
        self.j = None

    def A(self):
        self.i = ExternalClass.External()
        return self.i

    def B(self):
        self.j = ExternalClass.External()
        return self.j

File: testcases/paper/FilterMethodCalls/FilterMethodCalls.go
package FilterMethodCalls

type ExternalClass struct {
    external int
}

func (ec *ExternalClass) External() int {
    return 5
}

var ExternalClassInstance ExternalClass

type FilterMethodCalls struct {
    i, j int
}

func (fa *FilterMethodCalls) A() int {
    fa.i = ExternalClassInstance.External()
    return fa.i
}

func (fa *FilterMethodCalls) B() int {
    fa.j = ExternalClassInstance.External()
    return fa.j
}
